### DISCUSSION TITLE
How to install on Raspberry bookworm?

### DISCUSSION THREAD

**Discussion by sugramm** [goastian/midori-desktop#82] [General]
Hello

How can i install the Midori Browser on my RPI4 with bookworm?
I get an error Message by sudo wget -O /etc/apt/trusted.gpg.d/midori-archive-keyring.gpg http://repo.astian.org/midori-archive-keyring.gpg in the Terminal.
`--2024-08-18 12:51:56--  http://repo.astian.org/midori-archive-keyring.gpg
Resolving repo.astian.org (repo.astian.org)... failed: Name or service not known.
wget: unable to resolve host address ‘repo.astian.org’`

**Comment by Ponchale** [comment]
https://github.com/goastian/midori-desktop/releases/

We will be releasing the new version soon, please download the .deb file.
